ush/config_defaults.yaml

```diff
@@ -565,13 +565,7 @@ workflow:
   # a "ESGgrid" type of regional grid.
   #
   # FV3_NML_FN: 
-  # Name of the forecast model's namelist file. It includes the information in 
-  # FV3_NML_BASE_SUITE_FN (i.e., input.nml.FV3),  FV3_NML_YAML_CONFIG_FN (i.e., FV3.input.yml), and the user configuration file (i.e., config.yaml).
-  #
-  # FV3_NML_BASE_SUITE_FN:
-  # Name of Fortran namelist file containing the forecast model's base suite
-  # namelist, i.e. the portion of the namelist that is common to all physics
-  # suites.
+  # Name of the forecast model's namelist file.
```

Comment:
SRW puts a base namelist file (or set of files) in the experiment directory. With the UW Driver, there is no need to do this, so I have removed a lot of the references to that staged namelist.